### ISSUE TITLE
Minor Code Quality issues need to be reviewed and addressed

### ISSUE THREAD

**Issue by datapharmer** [financialforcedev/apex-mdapi#222]
Per a review with SonarQube a number of minor code quality issues exist that should be reviewed and addressed such as duplicate declaration of literals, public accessibility of class variables and unused private fields
[code quality issues.xlsx](https://github.com/financialforcedev/apex-mdapi/files/2402205/code.quality.issues.xlsx)

**Comment by afawcett** [comment]
These enhancements once addressed might address this https://github.com/financialforcedev/apex-mdapi/issues/84?
